self-hosting/README.md

```diff
@@ -0,0 +1,302 @@
+# HOSTING THE CHT APP
+
+The community health toolkit core framework has been packaged into a docker container to make it portable and easy to install. It is available from [dockerhub](https://hub.docker.com/r/medicmobile/medic-os). To learn more how to work with docker you could follow the tutorial [here](https://docker-curriculum.com/#getting-started) and the cheat sheet [here](https://www.docker.com/sites/default/files/d8/2019-09/docker-cheat-sheet.pdf).  
+
+## Installing Docker Prerequisites
+
+This Ideally depends on which operating system you are running. You need to first install these on your box before you can run the CHT docker containers.
+
+### 64-bit Linux
+
+Depending on which distro you run, install the docker packages from [here](https://docs.docker.com/engine/install/). We have however, historically run most of our containers in Ubuntu.
+
+#### 64-bit Ubuntu
+
+- [Docker CE](https://docs.docker.com/engine/install/ubuntu/)  
+
+- [Docker-compose](https://docs.docker.com/compose/install/)
+
+### Windows 
+
+The CHT docker image was built in Ubuntu so for it to work on windows you need to install The Windows Subsystem for Linux (WSL) in addition to docker desktop.
```

Comment:
WSL is not required to get docker-compose to work on windows.  Once docker-compose for windows is installed the commands are available for use in windows terminal just as if you were running on linux. Copy the yml to a directory and then run `docker-compose up`